mixer/test/client/pilotplugin/pilotplugin_test.go

```diff
@@ -164,7 +164,7 @@ static_resources:
   "destination.service.host": "svc.ns3",
   "destination.service.name": "svc",
   "destination.service.namespace": "ns3",
-  "destination.service.uid": "svcuid",
+  "destination.service.uid": "istio://ns3/services/svc",
```

Comment:
bug...

Reply:
What do you mean ? Should it return svcuid ??? There is no code generating this form

Reply:
UID is set in the test. It's an opaque string.

Reply:
The test was assuming a particular implementation and testing its own mock.